### DISCUSSION TITLE
Error - verify my academy status

### DISCUSSION THREAD

**Discussion by VincentSan** [community/community#53552] [GitHub Education]
Hi, 

I am currently a student at CNAM, and I have been trying to verify my academy status with GitHub's automated system. Unfortunately, I have been encountering some difficulties with the process, and I am not able to confirm my academy status through this system. I am reaching out to this forum to ask if anyone knows of an alternative way to verify my academy status on GitHub. I have been searching for a solution to this problem for some time now, but I have not been able to find any information that has been helpful to me.

If anyone has any tips or advice. 😊

**Comment by samus-aran** [comment]
Hi @VincentSan  👋
We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by VincentSan** [reply]
Hi @samus-aran 👋,
I have 2 rejection reasons : 

- "You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio."
- "Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy."

I completed both but nothing works properly i really don't know what to do.
